packages/url-loader/src/constants/parameters.ts

```diff
@@ -0,0 +1,223 @@
+import { z } from 'zod';
```

Comment:
question: This looks really clean but the question is whether it is enough amount of parameters? I think Cloudinary provides more so I am not sure if they are missing or delivered in a different place?

Reply:
great question - this is only a starting point, im trying to balance getting this shipped which includes a lot of great stuff and being comprehensive about it which is really really challenging. there's still a lot of types to fill in and such and if you and @matiasfha are interested in helping with we can discuss as those can be done as time goes on

any opposition to that? as i / we work on cleaning up the types and all my hope is to fill this in with more parameters to create a single source for these definitions as they're used throughout the project

Reply:
Count with me too